config/commands.js

```diff
@@ -385,7 +385,7 @@ var commands = exports.commands = {
 				if (move.flags['sound']) details["<font color=black>&#10003; Sound</font>"] = "";
 				if (move.flags['bullet']) details["<font color=black>&#10003; Bullet</font>"] = "";
 				if (move.flags['pulse']) details["<font color=black>&#10003; Pulse</font>"] = "";
-				if (move.flags['protect']) details["<font color=black>&#10003; Blocked by Protect</font>"] = "";
+				if (!move.flags['protect'] && !/(ally|self)/i.test(move.target)) details["<font color=black>&#10003; Ignores Protect</font>"] = "";
```

Comment:
I also think "Bypasses Protect" sounds nice, however  /details also returns "Ignores Substitute". I think we should either make them both 'bypass' or both 'ignore'.